src/api/app/components/notification_component.rb

```diff
@@ -13,7 +13,11 @@ def notification_icon
     when 'Comment'
       tag.i(class: ['fas', 'fa-comments'], title: 'Comment notification')
     else
-      tag.i(class: ['fas', 'fa-user-tag'], title: 'Relationship notification')
+      if @notification.event_type == 'Event::BuildFail'
+        tag.i(class: ['fas', 'fa-times-circle'], title: 'Package notification')
```

Comment:
We already use this icon for deleting content, so I suggest using another icon. The various icons for build statuses are defined in a helper:

https://github.com/openSUSE/open-build-service/blob/842c83e03dc0e48516d68595a296ae2dfae0ee0c/src/api/app/helpers/webui/buildresult_helper.rb#L2-L18